<!DOCTYPE html>
<html>

  <head>
    <meta charset='utf-8' />
    <meta http-equiv="X-UA-Compatible" content="chrome=1" />
    <meta name="description" content="daisieh/phylogenomics : (mostly) perl scripts that do something related to phylogenomic analysis." />

    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">

    <title>daisieh/phylogenomics</title>
  </head>

  <body>

    <!-- HEADER -->
    <div id="header_wrap" class="outer">
        <header class="inner">
          <a id="forkme_banner" href="https://github.com/daisieh/phylogenomics">View on GitHub</a>

          <h1 id="project_title">daisieh/phylogenomics</h1>
          <h2 id="project_tagline">(mostly) perl scripts that do something related to phylogenomic analysis.</h2>

            <section id="downloads">
              <a class="zip_download_link" href="https://github.com/daisieh/phylogenomics/zipball/master">Download this project as a .zip file</a>
              <a class="tar_download_link" href="https://github.com/daisieh/phylogenomics/tarball/master">Download this project as a tar.gz file</a>
            </section>
        </header>
    </div>

    <!-- MAIN CONTENT -->
    <div id="main_content_wrap" class="outer">
      <section id="main_content" class="inner">
        <p>This repo contains visualization and analysis scripts that I've written to help me perform phylogenomic analyses. Almost all are written in Perl. Some require the <a href="https://github.com/bioperl">Bioperl</a> package.</p>

<p>Scripts that require the <a href="https://github.com/bioperl">Bioperl</a> package will either call the Bio package directly or require subfunctions that are in the file bioperl_subfuncs.pl. It might be easiest if you include this directory in your $PERL5LIB path so that the various subfunction files are easy for the interpreter to find.</p>

<hr><h3>
<a name="files" class="anchor" href="#files"><span class="octicon octicon-link"></span></a>Files:</h3>

<p><strong>CircleGraph.pm</strong> is an object-oriented package that makes it easier to draw graphs along a circle. It makes postscript objects and uses the Postscript::Simple package.</p>

<p><strong>subfuncs.pl</strong> is a random hodgepodge of utility functions. As I write more of them, I may move some to more specific subfunction files to reduce overhead.</p>

<p><strong>bioperl_subfuncs.pl</strong> has helper functions that require <a href="https://github.com/bioperl">Bioperl</a> packages.</p>

<p><strong>circlegraph.pl</strong> has helper functions that draw specific kinds of circular graphs.</p>

<p>The subdirs contain scripts for various tasks:</p>

<ul>
<li>
<strong>analysis:</strong> scripts that perform various analyses on datasets.

<ul>
<li>    introgression: scripts to calculate statistics for introgression analysis.</li>
<li>    selection: scripts that use <a href="https://github.com/bioperl">Bioperl</a>, <a href="http://abacus.gene.ucl.ac.uk/software/paml.html">PAML</a>, <a href="http://hyphy.org/w/index.php/Main_Page">HyPhy</a> to analyze selection on genes.</li>
</ul>
</li>
<li>
<strong>converting:</strong> scripts to convert files from one format to another or to combine files in various ways.</li>
<li>
<strong>filtering:</strong> scripts that reduce complexity.</li>
<li>
<strong>parsing:</strong> uses <a href="https://github.com/bioperl">Bioperl</a> to slice and dice sequence files in various ways.</li>
<li>
<strong>plastome:</strong> scripts that take input about the plastome and draw circle graphs to represent that data.</li>
<li>
<strong>reporting:</strong> scripts that take vcf or sequence files and report statistics about them.</li>
</ul><hr><h3>
<a name="versions" class="anchor" href="#versions"><span class="octicon octicon-link"></span></a>Versions:</h3>

<p><strong>v1.0</strong> (<a href="https://github.com/daisieh/phylogenomics/archive/v1.0.zip">zip archive</a>)(<a href="https://github.com/daisieh/phylogenomics/archive/v1.0.tar.gz">tar archive</a>) is the version used in DI Huang et al, "Whole plastome sequencing reveals deep plastid divergence and cytonuclear discordance between closely related balsam poplars, Populus balsamifera and P. trichocarpa (Salicaceae)" (submitted).</p>
      </section>
    </div>

    <!-- FOOTER  -->
    <div id="footer_wrap" class="outer">
      <footer class="inner">
        <p class="copyright">daisieh/phylogenomics maintained by <a href="https://github.com/daisieh">daisieh</a></p>
        <p>Published with <a href="http://pages.github.com">GitHub Pages</a></p>
      </footer>
    </div>

    

  </body>
</html>
