<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <link href='https://fonts.googleapis.com/css?family=Chivo:900' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen" />
    <link rel="stylesheet" type="text/css" href="stylesheets/pygment_trac.css" media="screen" />
    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print" />
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <title>Phylogenomics by daisieh</title>
  </head>

  <body>
    <div id="container">
      <div class="inner">

        <header>
          <h1>Phylogenomics</h1>
          <h2>(Mostly) Perl scripts that do something related to phylogenomic analysis.</h2>
        </header>

        <section id="downloads" class="clearfix">
          <a href="https://github.com/daisieh/phylogenomics/zipball/master" id="download-zip" class="button"><span>Download .zip</span></a>
          <a href="https://github.com/daisieh/phylogenomics/tarball/master" id="download-tar-gz" class="button"><span>Download .tar.gz</span></a>
          <a href="https://github.com/daisieh/phylogenomics" id="view-on-github" class="button"><span>View on GitHub</span></a>
        </section>

        <hr>

        <section id="main_content">
          <p>This repo contains visualization and analysis scripts that I've written to help me perform
phylogenomic analyses. Almost all are written in Perl. Some require the Bioperl package.</p>

<p>Scripts that require the Bioperl package will either call the Bio package directly or require
subfunctions that are in the file bioperl_subfuncs.pl. It might be easiest if you include
this directory in your $PERL5LIB path so that the various subfunction files are easy for
the interpreter to find.</p>

<p><strong>CircleGraph.pm</strong> is an object-oriented package that makes it easier to draw graphs along
a circle. It makes postscript objects and uses the Postscript::Simple package.</p>

<p><strong>subfuncs.pl</strong> is a random hodgepodge of utility functions. As I write more of them, I may
move some to more specific subfunction files to reduce overhead.</p>

<p><strong>bioperl_subfuncs.pl</strong> has helper functions that require Bioperl packages.</p>

<p><strong>circlegraph.pl</strong> has helper functions that draw specific kinds of circular graphs.</p>

<p>The subdirs contain scripts for various tasks:</p>

<ul>
<li>
<strong>analysis:</strong> scripts that perform various analyses on datasets.

<ul>
<li>    introgression: scripts to calculate statistics for introgression analysis.</li>
<li>    selection: scripts that use Bioperl, PAML, HyPhy to analyze selection on genes.</li>
</ul>
</li>
<li>
<strong>converting:</strong> scripts to convert files from one format to another or to combine files in various ways.</li>
<li>
<strong>filtering:</strong> scripts that reduce complexity.</li>
<li>
<strong>parsing:</strong> uses Bioperl to slice and dice sequence files in various ways.</li>
<li>
<strong>plastome:</strong> scripts that take input about the plastome and draw circle graphs to represent that data.</li>
<li>
<strong>reporting:</strong> scripts that take vcf or sequence files and report statistics about them.</li>
</ul>
        </section>

        <footer>
          Phylogenomics is maintained by <a href="https://github.com/daisieh">daisieh</a><br>
          This page was generated by <a href="http://pages.github.com">GitHub Pages</a>. Tactile theme by <a href="https://twitter.com/jasonlong">Jason Long</a>.
        </footer>

        
      </div>
    </div>
  </body>
</html>